src/main/java/Marites.java

```diff
@@ -0,0 +1,198 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+import static java.lang.Integer.parseInt;
+
+public class Marites {
+
+    /* STRING CONSTANTS */
+
+    // Logo found in https://emojicombos.com/kaomoji
+    private static final String INTRO_MESSAGE = "(งツ)ว\n" +
+            "Hi, I'm Marites! I've heard so many things about you!\n" +
+            "I have a lot of stories to share, but first, how can I help you?\n";
+    private static final String EXIT_MESSAGE = "See you next time!";
+    private static final String UNKNOWN_COMMAND_MESSAGE = "I didn't understand that.";
+
+    private static final String COMMAND_EXIT = "bye";
+    private static final String COMMAND_LIST = "list";
+    private static final String COMMAND_MARK = "mark";
+    private static final String COMMAND_UNMARK = "unmark";
+    private static final String COMMAND_ADD_TODO = "todo";
+    private static final String COMMAND_ADD_DEADLINE = "deadline";
+    private static final String COMMAND_ADD_EVENT = "event";
+    private static final String COMMAND_ADD_DEADLINE_TAG = "/by";
+    private static final String COMMAND_ADD_EVENT_TAG = "/at";
+
+    private static final String LIST_TASK_ITEM_FORMAT_STRING = "%d. %s%n";
```

Comment:
Maybe you might not need a long constant to check for these format strings. Additionally, could be using too many constants